python-sdk/nuimages/nuimages.py

```diff
@@ -701,52 +794,58 @@ def render_image(self,
         im = Image.open(im_path)
 
         # Initialize drawing.
-        font = ImageFont.load_default()
+        font = get_font(fonts_valid=['FreeSerif.ttf', 'FreeSans.ttf'], font_size=font_size)
```

Comment:
I think drawing in opencv is a better choice if we want to avoid the font issue.

Reply:
@holger-nutonomy tried this approach before and his findings were:
```
1. I tried this. CV2 does not allow masks. The only thing I found is to compute a weighted average of two images with alpha channel, but it sounds excessive to copy an entire image, rather than a changing only the pixels of a small mask. You could give it a try though.
```
We have an ongoing thread [here](https://docs.google.com/document/d/1LbCam0YscbADJS8oibv4YrkpCJEsisRkBmeZxBgWtlc/edit?disco=AAAAGuEOHpk) and are trying out option 3 at the moment:
```
As an option 3: we can find a list of fonts present in the user's system using matplotlib's findSystemFonts method (https://matplotlib.org/api/font_manager_api.html). However, the downside will be that we can't control what font is used.
```
If you happen to have ideas on how this can be done better, do let us know :)